source/puppet/4.9/style_guide.markdown

```diff
@@ -412,10 +412,10 @@ file {
     * => $defaults ;
 
   '/tmp/motd' :
-   content => "message of the day" ;
+   content => 'message of the day' ;
```

Comment:
I dislike the space before the `;`. This occurs often and is non-standard, though has no official guideline that I saw. This line should be `'message of the day',;` due to trailing-commas must as well.